manifests/0000_50_cluster-monitoring-operator_05-deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: openshift-monitoring
   labels:
     app: cluster-monitoring-operator
```

Comment:
it would be good to remove this label while we're at it

Reply:
The selector at line 13-15 is immutable during upgrade, matching label `app: cluster-monitoring-operator`.
If we remove this label at line 7 and line 19, the CMO deploy may not find any pods to manage. So I leave both app labels where they are 😅
Test with removing them has been done with [this commit](https://github.com/openshift/cluster-monitoring-operator/pull/1388/files/40f7f8803dc24c752619c4abd65aa7df8577eea7#diff-984d250e7b24f523400c14314267a47c0c8fa79c62f07bef798af2c462f91777). The upgrade test did not pass.